model/account.rb

```diff
@@ -7,6 +7,10 @@ class Account < Sequel::Model(:accounts)
   include Authorization::HyperTagMethods
 
   def hyper_tag_name(project = nil)
+    ubid
+  end
+
+  def to_policy(project = nil)
```

Comment:
Do we need project here?

Reply:
to_policy_text or to_policy_label (i.e. something that doesn't refer to the whole policy) maybe?

Reply:
actually, we don't use "to_" prefix in hyper_tag_name. Maybe just policy_test/policy_label/policy_something?